### PR TITLE
add libsolv to the CI + simplify testing librepo

### PR DESCRIPTION
**Pull request by ignatenkobrain**

### CODE REVIEW

**Comment by ignatenkobrain** [comment]
librepo requires: https://bugzilla.redhat.com/show_bug.cgi?id=1324840

**Comment by jsilhan** [comment]
LGTM, @j-mracek if you have no objections, then please merge it.

**Comment by jsilhan** [comment]
(it shoudl be tested)

File: jenkins/ci-dnf-stack.yaml [hunk 1]
```diff
@@ -0,0 +1,132 @@
+- project:
```
Comment (j-mracek) [inline jenkins/ci-dnf-stack.yaml]: What the purpose of this file?

Comment (ignatenkobrain) [inline jenkins/ci-dnf-stack.yaml]: to maintain jenkins jobs

Comment (j-mracek) [inline jenkins/ci-dnf-stack.yaml]: Please can you explain, how to use it in readme.md file for others who would like to use our project, or for future work. Short description with external link also should work.

File: ci_dnf_stack/utils.py [hunk 2]
```diff
@@ -28,7 +28,7 @@ def _fixup_spec(package, spec):
     if package == "libsolv":
         tmp.insert(tmp.index("%check") + 1, "chmod +x test/runtestcases")
     elif package == "librepo":
-        tmp.insert(tmp.index("%check") + 1, "find tests/ -type f -name '*.sh' -exec chmod +x {} ';'")
+        tmp.insert(tmp.index("%check") + 1, "find tests/ -type f -name '*.sh' -or -name '*.sh.in' -exec chmod +x {} ';'")
```
Comment (j-mracek) [inline ci_dnf_stack/utils.py]: Even with this flag, the build of librepo doesn't work.

Comment (ignatenkobrain) [inline ci_dnf_stack/utils.py]: fixed. https://copr.fedorainfracloud.org/coprs/rpmsoftwaremanagement/dnf-pull-requests/build/174976/

Comment (ignatenkobrain) [inline ci_dnf_stack/utils.py]: py3 broken..... argh!

**Comment by ignatenkobrain** [comment]
now finally should be ok.

https://copr.fedorainfracloud.org/coprs/rpmsoftwaremanagement/dnf-pull-requests/build/174977/

**Comment by j-mracek** [comment]
Merged, thank you

**Comment by j-mracek** [comment]
Closed

**Pull request merged**